homeassistant/components/stookalert/binary_sensor.py

```diff
@@ -0,0 +1,76 @@
+"""This component provides support for Stookalert Binary Sensor."""
+import logging
+import voluptuous as vol
+import stookalert
+from datetime import timedelta
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, STATE_OFF, STATE_ON
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+CONF_PROVINCE = "province"
+DEFAULT_NAME = "Stookalert"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+		vol.Required(CONF_PROVINCE): vol.In([
+            "drenthe", "flevoland", "friesland", "gelderland", "groningen", "limburg", 
+            "noord-brabant", "noord-holland", "overijssel", "utrecht", "zeeland", "zuid-holland"]),
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    })
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the binary sensor platform"""
+    add_devices([Stookalert(config)], update_before_add=True)
+
+
+class Stookalert(Entity):
+    """An implementation of RIVM Stookalert"""
+
+    def __init__(self, config):
+        """Initialize a Stookalert device."""
+        self._province = config.get(CONF_PROVINCE)
+        self._apiHandler = stookalert.stookalert(self._province)
```

Comment:
`self._apiHandler` needs to be renamed.

```suggestion
        self._api_handler = stookalert.stookalert(self._province)
```